cumulus/parachains/runtimes/coretime/coretime-rococo/src/lib.rs

```diff
@@ -129,7 +129,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("coretime-rococo"),
 	impl_name: create_runtime_str!("coretime-rococo"),
 	authoring_version: 1,
-	spec_version: 1_005_004,
+	spec_version: 1_006_001,
```

Comment:
Why is it 1.6.1?

Reply:
when we did the relay chain upgrade, the migration failed and so we had to bump

Reply:
Did it work the second time or will we have to bump again?

Reply:
It worked, we have it running and configured on rococo and the sales have started (with 0 cores for this sale period).